packages/babel-plugin-proposal-pipeline-operator/test/fixtures/smart/topic-inside-arrow-function/input.js

```diff
@@ -0,0 +1,5 @@
+const result = -2.2   // -2.2
+  |> Math.floor       // -3
+  |> (() => Math.pow(#, 5))   // () => -243
```

Comment:
@js-choi Question on the spec behavior: if we didn't use a topic token here, this arrow would get executed?

```js
-2.2
    |> (x => Math.pow(x, 5))
```

This is also valid in Smart. Are we checking whether the token appears in the arrow body?